pyproject.toml

```
@@ -35,7 +35,7 @@ test = [
     "pytest >=4.3.0",
     "flake8",
     "flake8-bugbear",
-    "black",
+    "black; implementation_name=='pypy'",
```

Now that is embarrassing...

```suggestion
    "black; implementation_name=='cpython'",
```